frontend/src/hocs/withScreencast/withScreencast.tsx

```diff
@@ -123,14 +119,17 @@ function withScreencast(
       let outputBlob
       const { currentState } = this.state
 
+      if (currentState === "OFF") {
```

Comment:
Why is this needed? Is it to avoid line 128 to show an error when state is "OFF"?

If so, just move the OFF check inside the `if` from line 126.

Reply:
If the user press stop recording when the screencast is off , we should do nothing otherwise we we'll throw an exception to the frontend